rabbitmq/src/main/java/io/micronaut/rabbitmq/RabbitHeaders.java

```diff
@@ -40,9 +40,9 @@
     private final Map<String, String> headers;
 
     /**
-     * Constructs a new instance for the given headers.
+     * Constructs a new instance for the given map of headers.
      *
-     * @param headers The kafka headers
+     * @param headers A Map of headers
      */
     public RabbitHeaders(Map headers) {
```

Comment:
This map should be parameterized, `Map<String, String>` or perhaps `Map<String, Object>`

Reply:
Changed to Map<String, Object>